exercises/practice/house/house_test.go

```diff
@@ -112,9 +113,11 @@ that lay in the house that Jack built.`
 
 func TestVerse(t *testing.T) {
 	for v := 0; v < len(expectedVerses); v++ {
-		if ret := Verse(v + 1); ret != expectedVerses[v] {
-			t.Fatalf("Verse(%d) =\n%q\n  want:\n%q", v+1, ret, expectedVerses[v])
-		}
+		t.Run(strconv.Itoa(v+1), func(t *testing.T) {
```

Comment:
Similar suggestions to #2435

Reply:
To be honest I don't think there is a need of adding a struct and testcases, I have simply added `verse` to the description, what do you think?